### ISSUE TITLE
Remove config debug log

### ISSUE THREAD

**Issue by stefanos82** [Fushko/gammy#78]
The title is rather clear; how can we disable the debug option?

A `gammylog.txt` is found in `$HOME` with the following info in it:

```
2021-01-14 18:40:26.805 DEBUG [8332] [config::read@92] Reading from: /home/stefanos/.config/gammyconf
```
Shouldn't this output be enabled via `make debug` or with a gammy flag during initial execution time?

**Comment by Fushko** [comment]
Hi, that line gets outputted because the log_level is "debug" before reading the config in order to troubleshoot #46. It appears to have been fixed so I'll change it for the next release.

**Comment by stefanos82** [comment]
I would suggest to add a `make debug` option and let `make` to behave like `make release`; this way we can debug any future problematic behavior by rebuilding it in the aforementioned methodology.

Of course, if you would like you could add a DEBUG flag as switch button in GUI or via CLI.

**Comment by Fushko** [comment]
The line has been moved to verbose level on `develop`.

> I would suggest to add a make debug option and let make to behave like make release; this way we can debug any future problematic behavior by rebuilding it in the aforementioned methodology.

I don't want people to recompile just to see some logs. By CLI is better.

**Comment by stefanos82** [comment]
I guess I will need a way to call it when I run it from my Application Finder (XFCE4), in case I need the debug option that is.

**Comment by Fushko** [comment]
Fixed on 0.9.63
